kafka_utils/kafka_rolling_restart/main.py

```diff
@@ -169,6 +176,22 @@ def get_broker_list(cluster_config):
         return [(id, data['host']) for id, data in brokers]
 
 
+def filter_broker_list(brokers, filter_by):
+    """Returns sorted list, a subset of elements from brokers in the form [(id, host)].
+    Passing empty list for filter_by will return all brokers.
```

Comment:
nit: This function would probably be easier to read if this was not applied. Filtering by an empty list should return an empty list, and checks should be done before hand 🙂

I see that you're calling this function only if a list of brokers has been given as input, so it's definitely fine to skip this condition